### PR TITLE
Create frontend

### PR DESCRIPTION
**Pull request by zozich**

### CODE REVIEW

File: frontend/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```
Comment (zozich) [inline frontend/README.md:3]: Please remove generic reamde

File: frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "frontend",
```
Comment (zozich) [inline frontend/package.json:2]: Ok here

**Review by zozich — COMMENTED** [review]
Please fix minor changes

File: frontend/.gitignore [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```
Comment (kor1k) [inline frontend/.gitignore:4]: **lorem**

**Review by kor1k — APPROVED** [review]
test change #1

**Pull request merged**